agent_guard_core/config/config_manager.py

```diff
@@ -0,0 +1,69 @@
+from enum import Enum
+from pathlib import Path
+
+from agent_guard_core.credentials.file_secrets_provider import FileSecretsProvider
+
+
+class ConfigurationOptions(Enum):
+    """
+    Enum for configuration keys used by Agent Guard.
+    """
+    SECRET_PROVIDER = "The secret provider that Agent Guard supports"
```

Comment:
What do you mean by the provider that Agent Guard supports? From my understanding, that's the parameter that determines which secret provider will be used. Am I wrong?